draft-ietf-rats-architecture.md

```diff
@@ -117,39 +119,40 @@ Appraisal Policy for Attestation Result:
 
 Attestation Result:
 
-: The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the results
+: The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the Evidence it has appraised
 
 Attester:
 
-: An entity whose attributes must be appraised in order to determine whether the entity is considered trustworthy, such as when deciding whether the entity is authorized to perform some operation
+: An entity (typically a device), whose Evidence must be appraised in order to determine whether it is considered trustworthy, such as when deciding whether it is authorized to perform some operation
```

Comment:
Is "it" referring to "Evidence" now?  That doesn't seem right.   Revert changes to "it".

Reply:
or "the Attester"